tutorials/README.md

```diff
@@ -1,5 +1,19 @@
-## Cortex Certifai Tutorials 
-This directory contains the self contained tutorials which walk through various parts of Certifai. 
+# Cortex Certifai Tutorials 
+This directory contains self contained tutorials which walk through various parts of Certifai *outside* of a notebook environment.
 
+### Bringing In Your Own Model CLI
+
+This tutorial continues from part 1 located in the *notebooks* folder. 
+
+This walkthrough will demonstrate how to package and expose a model as a web service using S2i. After that step is complete, the local model will be used to run a Certifai Scan through the CLI with the Scan Definition file.
+
+Prerequisites:
+
+* Certifai Toolkit 
+* Scan Definition file (from part 1)
+* A model of your choice
+* Train and Predict functions for the model
+
+### Documentation 
 These tutorials have docs to be followed along, which can be found at: https://cognitivescale.github.io/cortex-certifai/docs/about
```

Comment:
The `about` link seems an odd choice for getting people to the tutorial docs.

Reply:
It is a placeholder as there is no link to an overall tutorials or notebooks page in the docs